.env.example

```diff
@@ -30,6 +30,7 @@ CORE_JSON_RPC_HOST=127.0.0.1
 CORE_JSON_RPC_PORT=9998
 CORE_JSON_RPC_USERNAME=dashrpc
 CORE_JSON_RPC_PASSWORD=password
+CORE_NETWORK=regtest
```

Comment:
It's not just Core network, actually. It's the platform network as well. Let's call it just `NETWORK` like everywhere?